crates/bevy_window/src/window.rs

```diff
@@ -84,7 +84,14 @@ impl NormalizedWindowRef {
     }
 }
 
-/// Define how a window will be created and how it will behave.
+/// The defining [`Component`] for window entities,
+/// storing information about how it will appear and behave.
+///
+/// When the [`Window`] component is added to an entity, a new window will be opened.
+/// When it is removed or the entity is despawned, the window will close.
+///
+/// This component is syncronised with `winit` through `bevy_winit`:
```

Comment:
```suggestion
/// This component is synchronised with `winit` through `bevy_winit`:
```

Reply:
I realise it should be "synchronized" to follow american english convention too (which I think Bevy follows?) :woman_facepalming:

Reply:
Ah yep! I always screw that up as a Canadian. Missing an `h` regardless.

Reply:
Solved by https://github.com/bevyengine/bevy/pull/8858/commits/4adea2a97fd75e013fce9cdf118b3aa6edc8e8d3